ts/build_for_browser.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+rm -rf dist
+
+mv src/workspace.ts src/workspace.ts.bak
+mv src/node_wallet.ts src/node_wallet.ts.bak
+
+cp src/browser_shims.ts src/workspace.ts
+cp src/browser_shims.ts src/node_wallet.ts
+
+yarn build
```

Comment:
this `yarn build` could alternatively be replaced with `microbundle` but I do accept that it might be a bit too much of a hack right now 😅